account_move_template/static/description/index.html

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this change is not necessary as part of the migration. In fact, the change you reference (https://github.com/odoo/odoo/commit/8d06889) is present since v13 (https://github.com/odoo/odoo/pull/133900) so it doesn't seem to be something new in v17.